src/main/java/com/couchbase/lite/ManagerOptions.java

```diff
@@ -6,15 +6,22 @@
 public class ManagerOptions {
 
     /**
-     *  No modifications to databases are allowed.
+     * No modifications to databases are allowed.
      */
     private boolean readOnly = false;
 
     /**
-     *  automatically migrate blobstore filename
+     * automatically migrate blobstore filename
      */
     private boolean autoMigrateBlobStoreFilename = false;
 
+    /**
+     * the number of threads to keep in the pool for RemoteRequestExecutor
+     * ReplicationInternal.java: public static final int EXECUTOR_THREAD_POOL_SIZE = 5;
+     * https://github.com/couchbase/couchbase-lite-java-core/issues/343
+     */
+    private int executorThreadPooSize = -1;
```

Comment:
- thanks for catching my stupid spelling error.
- If 0 or lower, CBL use default value. 0 could be better as default number.